### DISCUSSION TITLE
0.207.20 was auto updated on my Steam install...

### DISCUSSION THREAD

**Discussion by pdxgeek421** [lloesche/valheim-server-docker#459] [General]
I really wish there was a way to prevent this in Steam... Now I can't play on my server until this gets updated. :P

**Comment by drpcken** [comment]
If they are running this docker image it should have automatically updated (mine did).  If the server you're referring to is running V+ you'll need to update the V+ mod in your local steam install.

**Reply by pdxgeek421** [reply]
My game install and V+ are fully updated.   The latest version of this docker image is from 11 days ago. I am not sure what you mean exactly.  A new image hasn't been released yet for today's update, unless you are seeing something I am not.

**Reply by deedoubledub** [reply]
A new image is not required for game updates. The image automatically updates the game unless you explicitly disabled `UPDATE_CRON`.

My server has automatically updated, but I haven't played on it to verify that there aren't other issues.

**Reply by drpcken** [reply]
This is exactly what happened to me. My server automatically updated the latest game update early yesterday morning.  All I had to do was update V+ on the server and then my local client.

**Reply by pdxgeek421** [reply]
It seems to have all sorted itself out overnight.
